criu/include/cr_options.h

```diff
@@ -174,6 +174,8 @@ struct cr_options {
 
 	/* This stores which method to use for file validation. */
 	int 			file_validation_method;
+
+	int			with_cpu_affinity;	/* restore cpu affinity */
```

Comment:
CRIU usually has comments in the line before and not in the same line after the code like here. Just move the the comment to the line above and drop the empty line.